snyk_eclipse_plugin/src/io/snyk/eclipse/plugin/runner/ProcessRunner.java

```diff
@@ -54,9 +65,19 @@ public ProcessBuilder createMacProcessBuilder(String command, Optional<String> p
 	}
 	
 	public ProcessBuilder createWinProcessBuilder(String command, Optional<String> path) {
-		if (command.startsWith("/")) command = command.replaceFirst("/", "");
-		ProcessBuilder pb = new ProcessBuilder("cmd.exe", "/c", command);
+		if (command.startsWith("\"/")) command = command.replaceFirst("/", "");
```

Comment:
Can't we use some "os detect" for this? Have a different _mode_ for path parsing/creation?

Reply:
no :(, if we run snyk with cmd.exe we have to care and figure out correct quotes.